plugin/evm/vm_upgrade_bytes_test.go

```diff
@@ -145,125 +144,128 @@ func TestVMUpgradeBytesPrecompile(t *testing.T) {
-	assert.ErrorContains(t, err, "mismatching SubnetEVM fork block timestamp in database")
-
-	// VM should not start if fork is moved forward
-	upgradeConfig.NetworkUpgrades.SubnetEVMTimestamp = big.NewInt(30)
-	upgradeBytesJSON, err = json.Marshal(upgradeConfig)
-	if err != nil {
-		t.Fatalf("could not marshal upgradeConfig to json: %s", err)
-	}
-	err = vm.Initialize(context.Background(), vm.ctx, dbManager, []byte(genesisJSONPreSubnetEVM), upgradeBytesJSON, []byte{}, issuer, []*common.Fx{}, appSender)
-	assert.ErrorContains(t, err, "mismatching SubnetEVM fork block timestamp in database")
-}
-
-func TestVMUpgradeBytesNetworkUpgradesWithGenesis(t *testing.T) {
-	// make genesis w/ fork at block 5
-	var genesis core.Genesis
-	if err := json.Unmarshal([]byte(genesisJSONPreSubnetEVM), &genesis); err != nil {
-		t.Fatalf("could not unmarshal genesis bytes: %s", err)
-	}
-	genesisSubnetEVMTimestamp := big.NewInt(5)
-	genesis.Config.SubnetEVMTimestamp = genesisSubnetEVMTimestamp
-	genesisBytes, err := json.Marshal(&genesis)
-	if err != nil {
-		t.Fatalf("could not unmarshal genesis bytes: %s", err)
-	}
-
-	// Get a json specifying a Network upgrade at genesis
-	// to apply as upgradeBytes.
-	subnetEVMTimestamp := time.Unix(10, 0)
-	upgradeConfig := &params.UpgradeConfig{
-		NetworkUpgrades: &params.NetworkUpgrades{
-			SubnetEVMTimestamp: big.NewInt(subnetEVMTimestamp.Unix()),
-		},
-	}
-	upgradeBytesJSON, err := json.Marshal(upgradeConfig)
-	if err != nil {
-		t.Fatalf("could not marshal upgradeConfig to json: %s", err)
-	}
-
-	// initialize the VM with these upgrade bytes
-	_, vm, _, _ := GenesisVM(t, true, string(genesisBytes), "", string(upgradeBytesJSON))
-
-	// verify upgrade is rescheduled
-	assert.False(t, vm.chainConfig.IsSubnetEVM(genesisSubnetEVMTimestamp))
-	assert.True(t, vm.chainConfig.IsSubnetEVM(big.NewInt(subnetEVMTimestamp.Unix())))
-
-	if err := vm.Shutdown(context.Background()); err != nil {
-		t.Fatal(err)
-	}
-
-	// abort a fork specified in genesis
-	upgradeConfig.NetworkUpgrades.SubnetEVMTimestamp = nil
-	upgradeBytesJSON, err = json.Marshal(upgradeConfig)
-	if err != nil {
-		t.Fatalf("could not marshal upgradeConfig to json: %s", err)
-	}
-
-	// initialize the VM with these upgrade bytes
-	_, vm, _, _ = GenesisVM(t, true, string(genesisBytes), "", string(upgradeBytesJSON))
-
-	// verify upgrade is aborted
-	assert.False(t, vm.chainConfig.IsSubnetEVM(genesisSubnetEVMTimestamp))
-	assert.False(t, vm.chainConfig.IsSubnetEVM(big.NewInt(subnetEVMTimestamp.Unix())))
-
-	if err := vm.Shutdown(context.Background()); err != nil {
-		t.Fatal(err)
-	}
-}
+// Commenting out the NetworkUpgrade Tests until we get a new NetworkUpgrade.
+// The existing and only NetworkUpgrade that exists is SubnetEVM Upgrade which is now required at genesis.
+
+// func TestVMUpgradeBytesNetworkUpgrades(t *testing.T) {
+// 	// Hack: registering metrics uses global variables, so we need to disable metrics here so that we can initialize the VM twice.
```

Comment:
Can we switch from commenting these out to using
```
t.Skip("no non-required network upgrades to test network upgrade abort")
```

Reply:
yes